cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -126,8 +149,18 @@ func (t *templateBuilder) buildNodeFromTemplate(mig *Mig, template *gce.Instance
 				return nil, err
 			}
 			node.Spec.Taints = append(node.Spec.Taints, kubeEnvTaints...)
+
+			if allocatable, err := t.buildAllocatable(node.Status.Capacity, *item.Value); err == nil {
+				nodeAllocatable = allocatable
+			}
 		}
 	}
+	if nodeAllocatable == nil {
+		glog.Warning("Could not extract kube-reserved from kubeEnv, setting allocatable to capacity.")
```

Comment:
Add information about the nodegroup for which we could not calculate allocatable

Reply:
Done